### DISCUSSION TITLE
List feature flags by creator

### DISCUSSION THREAD

**Discussion by daisy1754** [Unleash/unleash#7077] [Q&A]
Hi - we have many feature flags and we want to cleanup. In order to do so, I want to query feature by creator. It seems GET feature API doesn't provide `created_by` field. Is there a way to filter feature flags by creator?

**Comment by daisy1754** [comment]
I think I will do workaround by search API https://docs.getunleash.io/reference/api/unleash/search-events

**Comment by ivarconr** [comment]
Hi, thanks for your suggestion. We recently added created_by_user_id to all features being created. Our plans is to add this as a filter for the features search in the future.
